### PR TITLE
add new site stuff

### PR DESCRIPTION
**Pull request by cooldracula**
big change to the site, best to run hugo serve and check it out.

went for simpler setup in terms of content structure as well.

this will be work in progress, but good launching point i believe!

### CODE REVIEW

File: content/team/caleb-woodbine.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Document Title
```
Comment (BobyMCbobs) [inline content/team/caleb-woodbine.md:1]: duplicate file?

File: content/_index.org [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+#+title: Homepage
+#+image: /assets/ii-fresh.png
+#+image_alt_text: "the ii logo, bright yellow, filled with cartoon versions of software mascots from the various free software we use in our work."
+#+exclude_footer: true
 
-coming soon!
+* Hello!
+We are a tech company based in Aotearoa New Zealand. We develop and consult on
+open-source, cloudbased solutions that focus on collaboration. Specializing in
+cloud-native technology like Kubernetes, Talos, Knative, and more, our goal is
```
Comment (BobyMCbobs) [inline content/_index.org:9]: I'd suggest

> Cloud Native technologies like Kubernetes, Knative, Istio, Envoy and more ...

Although I quite like Talos, it's more on the periphery

File: content/_index.org [hunk 3]
```diff
@@ -1,3 +1,16 @@
-#+title: ii.nz
+#+title: Homepage
+#+image: /assets/ii-fresh.png
+#+image_alt_text: "the ii logo, bright yellow, filled with cartoon versions of software mascots from the various free software we use in our work."
+#+exclude_footer: true
 
-coming soon!
+* Hello!
+We are a tech company based in Aotearoa New Zealand. We develop and consult on
+open-source, cloudbased solutions that focus on collaboration. Specializing in
```
Comment (BobyMCbobs) [inline content/_index.org:8]: recommend

> cloud-based

File: content/_index.org (hunk 2 above)
Comment (cooldracula) [reply content/_index.org:9]: we write about it in our blog, so i think it's good to mention.

**Pull request merged**